### DISCUSSION TITLE
Lazyvim flash the default colour scheme for a split second before changing to Neosolarized

### DISCUSSION THREAD

**Discussion by clauseggers** [LazyVim/LazyVim#3440] [Q&A]
Neosolarized recently changed and is now requiring the colour scheme to be executed some place after loading the plugin. I have put this in `autocmds.lua`
```
vim.cmd.colorscheme("neosolarized")
```
but Lazyvim will flash the default colour scheme for a split second before changing to Neosolarized, and in some cases I will see remnants of the default colour scheme.

Is my command wrong, or in the wrong place?

**Comment by folke** [comment]
Did you read the LazyVim docs? It's in there. There's even a search function
